main.cpp

```diff
@@ -7,6 +7,11 @@
 #include "library/ecs/entity.h"
 #include "project/example-project-1/mushroom.h"
 
+//TEST
+#include <stdio.h>
+#include "helper/randomisation.h"
+//TEST END
+
```

Comment:
so you include the randomisation but didn't use anywhere ? :thinking:

Reply:
My bad, remainder of tests. I got rid of those lines of code now.